src/make_table_example.F90

```diff
@@ -950,7 +1035,7 @@ subroutine write_h5(filename,timestamp)
        cerror = cerror + error
     endif
     
-    if (doing_inelastic.or.doing_epannihil) then
+    if (doing_inelastic.or.doing_epannihil.or.doing_bremsstrahlung) then
```

Comment:
It seems like this will also write out the `IN_Nz` and `n_N_Ipoints` to  the .h5 file as long as `(doing_inelastic.or.doing_epannihil.or.doing_bremsstrahlung) .eqv. .true.`, but this is not correct.  it was fine before since inelastic and epannihil share independent variables (`Itable_temp` and `Itable_eta`)